### DISCUSSION TITLE
start wails dev  with browser only

### DISCUSSION THREAD

**Discussion by mgutz** [wailsapp/wails#1987] [Q&A]
I"m using a tiling windows manager on Linux. Whenever `wails dev` command restarts the app, the tiling window manager rearranges windows because of the new instance. It's a bit annoying as I'm constantly rearranging windows.  I thought `wails dev -browser` would be the solution, but that opens up BOTH the app and a browser tab. 

Is there a workaround to start wails dev with only the browser? I don't need to test the window until I do final  build. Most of the logic is in the JS web app.

**Comment by leaanthony** [comment]
This has been asked before. Please open a ticket if one doesn't exist
